### DISCUSSION TITLE
GraphQL errors every time content is updated

### DISCUSSION THREAD

**Discussion by natelandau** [requarks/wiki#3681] [Error / Bug Report]
**Describe the bug**
Up until a few days ago everything worked perfectly.  Now, any changes I make to content (either moving pages or editing them) results in an out-of-sync state with an error stating `GraphQL error: Cannot return null for non-nullable field ResponseStatus.slug`.
When I select the "conflict" option in the header to try to resolve the issue both the remote and the local version are identical.  However, the page displayed in my web browser does not reflect the changes made.

**To Reproduce**
1. Edit the content or metadata of any page
2. Press "save"

**Expected behavior**
The content should be saved without errors

**Screenshots**

**Host Info (please complete the following information):**
- Wiki.js v2.3.77 running in Docker on a Raspberry Pi (v2.3.81 doesn't seem to have an arm tag yet in docker hub)
- docker image: requarks/wiki:2.3-arm
- postgres database (v.9.6.17)
- Rendering engine: markdown
- Storage: local file system

**Comment by NGPixel** [comment]
You need to pull the latest `requarks/wiki:2.3-arm` image to run 2.3.81.

**Comment by natelandau** [comment]
every time I pull `requarks/wiki:2.3-arm` the version running in the container is v`2.3.77`.  Is it possible the arm image was never updated to the newest version?

**Comment by NGPixel** [comment]
The 2.3-arm docker manifest has been fixed and should now point to the 2.3.81 images. Can you try again?

**Comment by natelandau** [comment]
just tried pulling `requarks/wiki:2.3-arm` and I don't think it's right.  The logs say 
```
2020-05-18T14:54:36.619Z [MASTER] info: =======================================
2020-05-18T14:54:36.626Z [MASTER] info: = Wiki.js 2.0.0 =======================
2020-05-18T14:54:36.627Z [MASTER] info: =======================================
```
and the UI is flagged as a "Development Version" is the header.  In addition, going to the admin interface where it should show the version number, it says "Wiki.js n/a"

**Comment by windsIT** [comment]
- Edit a page and Press `save`，encounter the same problem：`GraphQL error: Cannot return null for non-nullable field ResponseStatus.slug.`
- When I select the "conflict" option in the header to try to resolve the issue both the remote and the local version are identical. However, the page displayed in my web browser does not reflect the changes made. (Source code was edited)
 - Just a page， its source code is about 4,000 lines. other page is OK.
